### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by b01001011**
Issue: #37 #38
- [x] Update to have 👎  policy.
- [x] Add `Issue Filing Convention`.
- [x] Review.
# Contributing to FunBot
## Issue Filing Convention
1. It is always welcomed to post an issue.
2. To get proceed, at least 3 approvals(👍) from the members are required.
## Pull Request Merging Convention
1. All the comments SHOULD be addressed prior to merging.
   - To express a refusal on merging, the reviewer SHOULD emoji 👎 on the description.
   - And release the 👎 emoji when solved so that at the merging, there SHOULD be no 👎 left. 
2. At least 3 reviewers SHOULD approve.
   - To express an approval, emoji 👍  on the description.
3. If any participant requests specific member for a review (eg. @masunghoon), hold until the member reviews the PR. 

FunProg Team

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 
+## Issue Filing Convention
+ 1. It is always welcomed to post an issue.
+ 2. To get proceed, at least 3 approvals(👍) from the members are required.
+
 ## Pull Request Merging Convention
  1. All the comments SHOULD to be addressed prior to merging.
- 2. At least 3 reviewers SHOULD approve 
-  - To express an approval, press 👍  on the decription.
+  - To express an objection, the reviewer SHOULD emoji 👎 on the description.
```
Comment (jwChung) [inline CONTRIBUTING.md]: - `objection` 의 의미가 얼마나 강하게 표현되는지 모르겠지만, -1을 질문이나 의견을 남길 경우 병합하지 마라는 의미로 전달되었으면 좋겠습니다 :)
- `SHOULD emoji` 사이에 동사가 없어도 되나요?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
 # Contributing to FunBot
 
+## Issue Filing Convention
+ 1. It is always welcomed to post an issue.
+ 2. To get proceed, at least 3 approvals(👍) from the members are required.
+
 ## Pull Request Merging Convention
  1. All the comments SHOULD to be addressed prior to merging.
```
Comment (jwChung) [inline CONTRIBUTING.md]: 질문: Should to be 이런 표현도 사용하나봐요. all the comments should be addressed 이 표현이 더 부자연스럽나요?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
- 2. At least 3 reviewers SHOULD approve 
-  - To express an approval, press 👍  on the decription.
+  - To express an objection, the reviewer SHOULD emoji 👎 on the description.
+  - And release the 👎 emoji when solved so that at the merging, there SHOULD be no 👎 left. 
+ 2. At least 3 reviewers SHOULD approve.
+  - To express an approval, emoji 👍  on the description.
  3. If any participant requests specific member for a review (eg. @masunghoon), hold until the member reviews the PR.
 
 Thanks!
```
Comment (jwChung) [inline CONTRIBUTING.md]: Thanks! 이거 떼도 되지 않을까요?

File: CONTRIBUTING.md (hunk 2 above)
Comment (b01001011) [inline CONTRIBUTING.md]: Eagle eye!!!! That's typo :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (b01001011) [inline CONTRIBUTING.md]: Yup, I will stress that point. 
And.. I have used emoji as a verb. It's not a formal way of writing though. :)

File: CONTRIBUTING.md (hunk 3 above)
Comment (b01001011) [inline CONTRIBUTING.md]: sure, why not.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
 
+## Issue Filing Convention
+ 1. It is always welcomed to post an issue.
+ 2. To get proceed, at least 3 approvals(👍) from the members are required.
+
 ## Pull Request Merging Convention
- 1. All the comments SHOULD to be addressed prior to merging.
- 2. At least 3 reviewers SHOULD approve 
-  - To express an approval, press 👍  on the decription.
+ 1. All the comments SHOULD to addressed prior to merging.
```
Comment (jwChung) [inline CONTRIBUTING.md]: SHOULD to 가 맞나요? ㅎ

Comment (b01001011) [inline CONTRIBUTING.md]: ㅎㅎㅎ

**Comment by jwChung** [comment]
There is no reason that I don't vote for +1. Thank you for your effort. :)

**Comment by b01001011** [comment]
@jwChung My pleasure! :)

**Pull request merged**